<launch>
  <rosparam file="$(find ghattas_control_advanced)/config/parameters.yaml" command="load"/>
  <node name="heading_control" pkg="ghattas_control_advanced" type="heading_control.py" output="screen"/>
  <node name="depth_control" pkg="ghattas_control_advanced" type="depth_control.py" output="screen"/>
  <node name="move" pkg="ghattas_control_advanced" type="move.py" output="screen"/>
  <node name="long_lat_control" pkg="ghattas_control_advanced" type="long_lat_control.py" output="screen"/>
</launch>
